src/OpenTelemetry.Api/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* **Breaking change:** Revert space character encoding change from `+` to `%20`
```

Comment:
Add this PR # as well.

Reply:
Added in https://github.com/open-telemetry/opentelemetry-dotnet/pull/5687/commits/e474fa78360f899aba3681697237d3e20f929f15

Reply:
Please add more information about why we're reverting this change to the changelog or PR description. Something to the effect of "... because the fix did not properly address the inconsistency with the baggage spec. We will address this issue more completely in a later release"

Reply:
PR description updated.